run_aprime.bash

```diff
@@ -527,6 +530,7 @@ if [ $atm_status -eq 0 ]    || [ $atm_status -eq -2 ]   ||
   done
   chmod ga+rX $www_dir
   chmod -R ga+rX $www_dir/$plots_dir_name
+  chmod -R ga+rX $mpas_www_dir
```

Comment:
If `$mpas_www_dir` gets put inside of `$www_dir/$plots_dir_name`, this extra chmod isn't necessary.

Reply:
As of now, I have `mpas_analysis/` sitting under `$www_dir` but *not* under `$www_dir/$plots_dir_name`. I could change that.